base/server/python/pki/server/deployment/__init__.py

```diff
@@ -366,14 +366,11 @@ def configure_kra(self, subsystem):
     def configure_tps(self, subsystem):
 
         baseDN = subsystem.config['internaldb.basedn']
-        dsHost = subsystem.config['internaldb.ldapconn.host']
-        dsPort = subsystem.config['internaldb.ldapconn.port']
```

Comment:
Are these parameters also to be removed? There's no mention of them in the commit message or PR description, the properties are still referenced elsewhere in the code and they are not removed by the upgrade script.

Reply:
No, here we're only loading `internaldb.ldapconn.*` into local vars, which then would be stored into `tokendb.hostport`. Since we're removing `tokendb.hostport`, we don't need to load `internaldb.ldapconn.*` into local vars anymore here. The `internaldb.ldapconn.*` params are still used somewhere else so we're not removing them.

Reply:
OK, thanks!